src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1559,6 +1553,18 @@ public String toVariableNameString(EnchantmentOffer eo) {
 			.name("Experience Cooldown Change Reason")
 			.description("Represents a change reason of an <a href='events.html#experience cooldown change event'>experience cooldown change event</a>.")
 			.since("INSERT VERSION"));
+
+		if (Skript.classExists("org.bukkit.entity.EntitySnapshot")) {
+			Classes.registerClass(new ClassInfo<>(EntitySnapshot.class, "entitysnapshot")
+				.user("entity ?snapshots?")
+				.name("Entity Snapshot")
+				.description("Represents a snapshot of an entity's data. ",
+					"This includes all of the data associated with an entity (it's name, health, attributes, etc.), at the time it's retrieved. ",
```

Comment:
```suggestion
					"This includes all of the data associated with an entity (its name, health, attributes, etc.), at the time this expression is used. "
					+ "Essentially, these are a way to create templates for entities.",
```

Reply:
this makes it very explicit that when it says "entity snapshot of ...", it'll pull the details. when "it's retrieved" sounds more ambiguous, so someone might assume that this means that it's retrieved a different time. not really that important, the `it's` -> `its` should remain though